rdrf/rdrf/templates/admin/patients/patientstage/change_form.html

```diff
@@ -0,0 +1,97 @@
+{% extends "rdrf_cdes/base.html" %}
```

Comment:
Can we extend from `admin/change_form.html` instead and add just the bit we need so we don't copy the whole file?

Reply:
Updated.